test/client.test.ts

```diff
@@ -2600,7 +2576,7 @@ describe('client', async () => {
         'data: {"listenerName":"LGFXwOqrf1GHawAjZRnhd6"}',
         '',
         'event: mutation',
-        `data: ${JSON.stringify({document: doc})}`,
+        `data: ${JSON.stringify({result: doc})}`,
```

Comment:
this aligns with mutation events from /listen endpoint returning `result`, not `document`